<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
<CodeBlocks_layout_file>
	<FileVersion major="1" minor="0" />
	<ActiveTarget name="debug" />
	<File name="src/forge/logging/LogContext.cpp" open="1" top="0" tabpos="5" split="0" active="1" splitpos="0" zoom_1="0" zoom_2="0">
		<Cursor>
			<Cursor1 position="0" topLine="0" />
		</Cursor>
	</File>
	<File name="src/test.cpp" open="1" top="0" tabpos="6" split="0" active="1" splitpos="0" zoom_1="0" zoom_2="0">
		<Cursor>
			<Cursor1 position="466" topLine="0" />
		</Cursor>
	</File>
	<File name="src/forge/exceptions/BaseException.cpp" open="1" top="0" tabpos="3" split="0" active="1" splitpos="0" zoom_1="0" zoom_2="0">
		<Cursor>
			<Cursor1 position="0" topLine="0" />
		</Cursor>
	</File>
	<File name="src/forge/logging.cpp" open="1" top="1" tabpos="4" split="0" active="1" splitpos="0" zoom_1="0" zoom_2="0">
		<Cursor>
			<Cursor1 position="440" topLine="0" />
		</Cursor>
	</File>
	<File name="src/forge/exceptions/BaseException.hpp" open="1" top="0" tabpos="7" split="0" active="1" splitpos="0" zoom_1="0" zoom_2="0">
		<Cursor>
			<Cursor1 position="127" topLine="0" />
		</Cursor>
	</File>
</CodeBlocks_layout_file>
